.gitignore

```diff
@@ -15,3 +15,6 @@ yarn-error.log
 
 # parceljs
 .parcel-cache
+
+# VSCode user-specific settings
+/.vscode
```

Comment:
Add line break for this part. Also is this intended? ``/.vscode`` ? It is supposed to be ``.vscode`` only.

Reply:
Ah. Sorry, a habit of mine. Somehow forgot those were dirs and not files. Looks like I got it everywhere, so I might have to change a few.